proto/dsmr/dsmr.proto

```diff
@@ -21,13 +21,7 @@ message GetChunkSignatureRequest {
 }
 
 message GetChunkSignatureResponse {
-  // TODO are these fields needed?
-  bytes chunk_id = 1;
-  bytes producer = 2;
-  int64 expiry = 3;
-
-  bytes signer = 4;
```

Comment:
These were in the original DSMR spec, but I don't think they're needed since the chunk metadata can be inferred from the request type (which requester knows), and the signer is known implicitly since the requester knows the node-id of who they're requesting from, so they can lookup the public key in the P-Chain.